GoogleSignIn/Sources/GIDGoogleUser.m

```diff
@@ -204,17 +292,23 @@ + (BOOL)supportsSecureCoding {
 - (nullable instancetype)initWithCoder:(NSCoder *)decoder {
   self = [super init];
   if (self) {
-    GIDProfileData *profileData =
-        [decoder decodeObjectOfClass:[GIDProfileData class] forKey:kProfileDataKey];
-    OIDAuthState *authState;
-    if ([decoder containsValueForKey:kAuthState]) { // Current encoding
-      authState = [decoder decodeObjectOfClass:[OIDAuthState class] forKey:kAuthState];
-    } else { // Old encoding
```

Comment:
We should try to maintain support for the old encoding.

Reply:
The old encoding decodes GIDAuthentication. Now we removed this class.

Reply:
So, one solution would be to define a `GIDAuthentication` class here for backward compatibility purposes that only implements `initWithCoder:`.  All you'd need to do is decode the `OIDAuthState` instance in `kAuthStateKey` and set the `authState` property.

Reply:
Done.